.github/workflows/testPackageOnWindows.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: Run packaging test
         run: |
           cd "${{ inputs.unreal-batch-files-path }}"
-          ./RunUAT.bat -ScriptsForProject="$env:TESTS_PROJECT_ROOT/TestsProject.uproject" Turnkey -command=VerifySdk -platform=Win64 -UpdateIfNeeded -EditorIO -EditorIOPort=54342  -project="$env:TESTS_PROJECT_ROOT/TestsProject.uproject" BuildCookRun -nop4 -utf8output -nocompileeditor -skipbuildeditor -$env:UNREAL_COOK_PARAM  -project="$env:TESTS_PROJECT_ROOT/TestsProject.uproject"  -unrealexe="${{ inputs.unreal-binaries-path }}/UnrealEditor-Cmd.exe" -platform=Win64 -ddc=InstalledDerivedDataBackendGraph -installed -stage -archive -package -build -compressed -iostore -pak -prereqs -archivedirectory="$env:GITHUB_WORKSPACE/TestPackaging" -nocompile
+          ./RunUAT.bat -ScriptsForProject="$env:TESTS_PROJECT_ROOT/TestsProject.uproject" Turnkey -command=VerifySdk -platform=Win64 -UpdateIfNeeded -EditorIO -EditorIOPort=54342  -project="$env:TESTS_PROJECT_ROOT/TestsProject.uproject" BuildCookRun -nop4 -utf8output -nocompileeditor -skipbuildeditor -$env:UNREAL_COOK_PARAM  -project="$env:TESTS_PROJECT_ROOT/TestsProject.uproject"  -unrealexe="${{ inputs.unreal-binaries-path }}/UnrealEditor-Cmd.exe" -platform=Win64 -ddc=InstalledDerivedDataBackendGraph -installed -stage -archive -package -build -compressed -iostore -pak -prereqs -archivedirectory="$env:GITHUB_WORKSPACE/TestPackaging" -clientconfig=Shipping -nodebuginfo -nocompile -nocompileuat
```

Comment:
Updated packaging script to what you'd see when packaging via the Unreal Editor, v5.3.1